.travis.yml

```diff
@@ -11,11 +15,17 @@ env:
   - WALLABY_DRIVER=chrome
   - WALLABY_DRIVER=selenium
 matrix:
-  exclude:
+  include:
+  - elixir: 1.5.3
+    otp_release: 20.3
+  - elixir: 1.6.6
+    otp_release: 20.3
+  - elixir: 1.7.4
+    otp_release: 21.3
```

Comment:
If you want to do `include` (which imo is a good idea!) I believe you shouldn't specify them further up under `elixir` and `otp_release` - not sure how that works together with our env variables but it's worth a shot.

Also: 1.5 should work with OTP 18, 1.6 with OTP 19 so that might be good to add to test that compatibility until we know they don't work (iirc some newer dialyzer or credo version might need OTP 19+ or even OTP 20+)

Reply:
It didn't work with our env variables really, it ran each env variable in default elixir container and then each elixir and otp included without env variables. I removed it and added all options with env variables to the matrix, it will get cleaned up a bit when we deprecate phantomjs.

I added OTP 18 and 19, we'll see if tests pass